recipes-core/systemd/systemd/verdin-am62-hmm/wait-for-network.service

```diff
@@ -0,0 +1,20 @@
+#  SPDX-License-Identifier: LGPL-2.1-or-later
+#
+#  This file is part of systemd.
+#
+#  systemd is free software; you can redistribute it and/or modify it
+#  under the terms of the GNU Lesser General Public License as published by
+#  the Free Software Foundation; either version 2.1 of the License, or
+#  (at your option) any later version.
```

Comment:
You can remove this text because your service file is not part of the systemd itself. You can keep the top SPDX-License Identifier though...